compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/Draggable.kt

```diff
@@ -235,9 +235,7 @@ internal class DraggableElement(
 
     override fun equals(other: Any?): Boolean {
         if (this === other) return true
-        if (javaClass != other?.javaClass) return false
-
-        other as DraggableElement
+        if (other !is DraggableElement) return false
```

Comment:
```suggestion
        if (other == null || this::class != other::class) return false
```
(IDEA generates this code)